cmd/bootstrap/runner.go

```diff
@@ -21,6 +24,13 @@ import (
 	"k8s.io/client-go/rest"
 )
 
+const (
+	appOperatorVersion                = "2.2.0"
+	chartOperatorVersion              = "2.3.1"
```

Comment:
I think it's better to explicitly set the operator versions. So we can be sure the combination is tested.

The new CLI is going to be used in the integration-test job in architect-orb.

Reply:
Another option is that you could use `GetLatestVersion` from appcatalog repo, but it's subject to change the latest version time by time for sure.

Reply:
Yes there are pros and cons but I think its better to pin the version and be sure we've tested the combination.

Otherwise if we use the latest and it causes tests to fail that is hard to debug.